crates/ruff/src/rules/ruff/snapshots/ruff__rules__ruff__tests__RUF200_bleach.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: crates/ruff/src/rules/ruff/mod.rs
+---
+pyproject.toml:5:16: RUF200 Failed to parse pyproject.toml: Version specifier `>=1.1.0<1.2` doesn't match PEP 440 rules
+tinycss2>=1.1.0<1.2
+        ^^^^^^^^^^^
+  |
+5 |   version = "0.1.0"
+6 |   # There's a comma missing here
+7 |   dependencies = [
+  |  ________________^
+8 | |   "tinycss2>=1.1.0<1.2",
+9 | | ]
```

Comment:
I assume this is a "bug" in the PEP440 crate. It should only return the range of the qualifier without leading and trailing whitespace.

Reply:
that's from the toml crate i think, the definition is at https://github.com/PyO3/pyproject-toml-rs/blob/e3ae25d4fa43c29351e5b6844b9428a0adb37d95/src/lib.rs#L63

Reply:
Oh, right. There's another crate :) Do you think it is because how `pyproject-toml-rs` creates the error or is it because the requirement range includes the whitespace too?